src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ManageContent/ManageContentPageService.cs

```diff
@@ -62,17 +62,22 @@ public class ManageContentPageService : IManageContentPageService
                 {
                     var (release, unattachedDataBlocks, files) = releaseBlocksAndFiles;
 
-                    var methodologyVersions = await _methodologyVersionRepository
-                        .GetLatestVersionByPublication(release.PublicationId);
-
-                    var approvedMethodologyVersions = methodologyVersions
-                        .Where(mv => mv.Approved)
-                        .ToList();
+                    List<MethodologyVersion> methodologyVersions;
+                    if (isPrerelease)
+                    {
+                        methodologyVersions = await _methodologyVersionRepository
+                            .GetLatestPublishedVersionByPublication(release.PublicationId);
+                    }
+                    else
+                    {
+                        methodologyVersions = await _methodologyVersionRepository
+                            .GetLatestVersionByPublication(release.PublicationId);
+                    }
 
                     var releaseViewModel = _mapper.Map<ManageContentPageViewModel.ReleaseViewModel>(release);
                     releaseViewModel.DownloadFiles = files.ToList();
                     releaseViewModel.Publication.Methodologies =
-                        _mapper.Map<List<IdTitleViewModel>>(approvedMethodologyVersions);
+                        _mapper.Map<List<IdTitleViewModel>>(methodologyVersions);
```

Comment:
If the version id set in the view model is important here, e.g. because pre-release users can click through to the content of it, then I think they are only ever going to see the content of the latest published version, which would exclude the  version of a methodology which is about to be published with this release.

I think pre-release users should see the methodologies list accurate to what it will be when the release is published. Might be one to check with Cam, on how this should work.

Reply:
Discussed this with Ben and Cam. So now this is the logic that determines what methodologies are displayed around Admin:
- On the Admin content pages: the latest methodology version is displayed, regardless of status.
- On Prerelease content and prerelease methodologies tab: the latest methodology version is displayed if it is approved, but if it is not approved, the previous version is displayed.

This should ensure that any methodology set to go out with a release will be displayed in prerelease. 

It's possible that a methodology set to be released with a different release may also appear on prerelease, but I think the time investment to ensure that is handled correctly, alongside the likelyhood of that scenario happening, makes that not worth it - especially as we have been displaying draft methodologies on prerelease for a long time and nobody complained..